package.json

```diff
@@ -68,7 +67,8 @@
     "text-buffer": "6.0.0",
     "theorist": "^1.0.2",
     "typescript-simple": "1.0.0",
-    "underscore-plus": "^1.6.6"
+    "underscore-plus": "^1.6.6",
+    "yargs": ">=3.7.2 <4.0"
```

Comment:
@kevinsawicki Can I change this back to a caret version range since carets are used everywhere else here?